grunt.js

```diff
@@ -314,6 +300,23 @@ module.exports = function(grunt) {
         src: ['build/temp/video.js', 'node_modules/vtt.js/dist/vtt.js'],
         dest: 'build/temp/video.js',
       },
+    },
+    concurrent: {
+      options: {
+        logConcurrentOutput: true
+      },
+      // Run multiple watch tasks in parallel
+      // Needed so watchify can cache intelligently
+      watchAll: [
+        'watch',
+        'browserify:watch',
```

Comment:
Also, this is why I separated them out into `test-ui` and `test-cli`, so you can choose which you're using. I like to let the cli version run and beep at me when there's test errors, so I'd personally prioritize `karma:watch` over the others.